src/algebra/order/chebyshev.lean

```diff
@@ -0,0 +1,123 @@
+/-
+Copyright (c) 2023 Mantas Bakšys, Yaël Dillies. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Mantas Bakšys, Yaël Dillies
+-/
+import algebra.big_operators.order
+import algebra.order.rearrangement
+import group_theory.perm.cycle.basic
+
+/-!
+# Chebyshev's sum inequality
+
+This file proves the Chebyshev sum inequality.
+
+Chebyshev's inequality states `(∑ i in s, f i) * ∑ i in s, g i ≤ s.card * ∑ i in s, f i * g i` when
+`f g : ι → α` monovary, and the reverse inequality when `f` and `g` antivary.
+
```

Comment:
Can you state a few results you prove here, with their names?

Reply:
Do you like it like that?